examples/hackers_delight/variants/p1/p1-dead.wat

```diff
@@ -0,0 +1,38 @@
+;; Turn-off rightmost 1 bit.
```

Comment:
@anirudhSK 

Added two no-op instruction to p1.wat and synthesized following 2 equivalent programs. 
Both equivalent programs also used nop instructions.